BDArmory/Control/MissileFire.cs

```diff
@@ -7554,7 +7576,13 @@ public void PointDefenseTurretFiring()
                     if (missile.Current == null) continue;
                     if (!missile.Current.engageMissile) continue;
                     if (missile.Current.HasFired || missile.Current.launched) continue;
-                    missileCount++;
+                    if (missile.Current.multiLauncher && missile.Current.multiLauncher.turret)
+                    {
+                        if (!missile.Current.multiLauncher.turret.turretEnabled) 
+                            missile.Current.multiLauncher.turret.EnableTurret(missile.Current);
+                        missileCount += (int)Mathf.Clamp(Mathf.Floor(missile.Current.multiLauncher.missileSpawner.ammoCount / missile.Current.multiLauncher.salvoSize), 1, missile.Current.multiLauncher.missileSpawner.ammoCount);
```

Comment:
Clamp was set to one to account for launchers that had ammo left, but ammoCount that was less than the salvo size so it would still regard an incomplete salvo as 1 missile.
I've added ``if (missile.Current.multiLauncher.missileSpawner.ammoCount == 0 && !BDArmorySettings.INFINITE_ORDINANCE) continue;`` at the start of the block to handle any launchers that are fully empty.

Reply:
Wouldn't `Mathf.Ceil` be better then? Or is there a reason you'd want both 0.5 salvoes and 1.5 salvoes worth of missiles to only count as 1?